bot/plotting/comparison_generator.py

```diff
@@ -6,6 +6,22 @@
 from experiment.experiment_generator import experiment_generator
 
 
+param_to_vary_dict = {
```

Comment:
Could you add a comment here explaining that the tuple is the bounds for each parameter